lib/analyze_hoc_activity_helper.rb

```diff
@@ -30,12 +30,25 @@ def analyze_day_fast(date)
     " WHERE (finished_at >= '#{day}' AND finished_at < '#{next_day}')" +
     "   OR (pixel_finished_at >= '#{day}' AND pixel_finished_at < '#{next_day}')"
 
+  # Generate a list of Code.org tutorials so that we can generate the count for
+  # Code.org hosted tutorials below.
+  codedotorg_tutorials = []
+  PEGASUS_REPORTING_DB_READONLY.fetch(
+    "SELECT code FROM tutorials WHERE orgname = 'Code.org'"
```

Comment:
I'm a little nervous that we're basing it off a text field that could easily have a typo. But... 

1) that gsheet is watched like a HAWK by Hadi, myself, etc.
2) since that field is displayed on /learn it's more likely to be caught early
3) doubtful, but possible that we could do something like "Code.org and Disney" but since we haven't yet it's not likely we will in the future.
4) even if there were a typo, it's not hard to fix and re-count.
5) there really isn't a better way we differentiate Code.org tutorial from not Code.org tutorial right now.

Just note to self as a point of improvement for the future.